docs/siem/detections/api/lists/api-import-list-items.asciidoc

```diff
@@ -0,0 +1,82 @@
+[[lists-api-import-list-items]]
+=== Import list items
+
+Imports a list of items from a `.txt` file.
+
+You can import items to a new or existing
+<<lists-api-create-container, list container>>.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/lists/items/_import`
+
+The request must include:
+
+* The `Content-Type: multipart/form-data` HTTP header.
+* A link to the file containing the list items.
+
+For example, using cURL:
+
+[source,console]
+--------------------------------------------------
+curl -X POST "<kibana host>:<port>/api/lists/items/_import"
+-u <username>:<password> -H 'kbn-xsrf: true'
+-H 'Content-Type: multipart/form-data'
+--form "file=@<link to file>" <1>
+--------------------------------------------------
+<1> The relative link to the file containing the list items.
+
+===== URL query parameters
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`list_id` |String |ID of the <<lists-api-create-container, list container>>.
+|Required when importing to an existing container.
+|`type` |String |The datatype of excludes the list container holds, which can
+be either `ip` or `keyword`. |Required when importing to a new container.
+|==============================================
+
+===== Example requests
+
+Adds the IP addresses in the `internal-IPs.txt` to the `internal-ip-excludes`
+list container:
+
+[source,console]
+--------------------------------------------------
+curl -X POST "api/lists/items/_import?list_id=internal-ip-excludes"
+-H 'kbn-xsrf: true' -H 'Content-Type: multipart/form-data'
+--form "file=@internal-IPs.txt"
+--------------------------------------------------
+
+Adds the IP addresses in the `internal-IPs.txt` to a new list container:
+
+[source,console]
+--------------------------------------------------
+curl -X POST "api/lists/items/_import?type=ip"
+-H 'kbn-xsrf: true' -H 'Content-Type: multipart/form-data'
+--form "file=@internal-IPs.txt"
+--------------------------------------------------
+
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
+    
+==== Response payload
+
+[source,json]
+--------------------------------------------------
+{
+  "id": "internal-ip-excludes",
+  "created_at": "2020-07-07T04:09:55.028Z",
+  "created_by": "Threat Hunter",
+  "description": "Contains list items that exclude internal IP addresses from detection rules.",
+  "name": "Exclude internal IP addresses",
+  "tie_breaker_id": "b123ab0e-d7c0-4ee5-9630-63a38641aa0c",
+  "type": "ip",
+  "updated_at": "2020-07-07T04:09:55.028Z",
+  "updated_by": "Threat Hunter"
+}
```

Comment:
This will technically return the same extra three things as mentioned above:

```ts
"_version": "Wzg1LDFd"
"version": 1
"immutable": false
```

But for the put/post we do not allow the version for OCC to be sent in. I would update the response but don't put any docs for the input being able to have those as that is just an oversight on my part here and a bug.

Once I fix that bug I will ping you and you can then update the docs to include it for the input as well.